setup.cfg

```diff
@@ -57,6 +57,7 @@ install_requires =
     PyYAML>=5.1
     pydantic>=1.8.1
     qtpy>=1.7.0
+    superqt
```

Comment:
Very minor nitpick, that should this list be sorted alphabetically?

Reply:
yes!  not minor! :)